docs/meta.rst

```diff
@@ -61,7 +61,7 @@ These functions can be used in both Taichi kernels and python scripts.
 For sparse tensors, the full domain shape will be returned.
 
 Compile-time evaluations
-------------------------------------------
+------------------------
 Using compile-time evaluation will allow certain computation to happen when kernels are instantiated.
```

Comment:
I'd say '' Using compile-time evaluation allows certain computations carried out during kernel initialization. This saves the overhead of those computations at runtime.''